### ISSUE TITLE
Is there a support for client POST with pure json data?

### ISSUE THREAD

**Issue by visibletrap** [http-kit/http-kit#128]
In the same way as this `clj-http` does with this snippet

``` clojure
(clj-http.client/post { :body "{\"json\": \"input\"}" })
```

Seems like neither `query-params` nor `form-params` can be used for that.

**Comment by limist** [comment]
Yes, agree with @liusong1111 that the docs should mention the common use-case of POST'ing JSON to an external API: that the data to post should be in the `:body` and not `:form-params`

**Comment by allen-servedio** [comment]
For anyone else that runs across this issue, you have to add a `Content-Type` header for this to work with most servers...

```
(http/post "url" {:body (json-str {:key "value", :key2 ["arr"]})
                  :headers {"Content-Type" "application/json"}})
```

**Comment by DarrenBishop** [comment]
Many years later, not great to have to find the solution here.

Propose you remove 
> Easy-to-use API, modeled after clj-http
as
```
(client/post "http://example.com/api"
             {...
              :body "{\"json\": \"input\"}"
              ...
              :content-type :json
              ...
              }
```
does not work, at least not for me

**Comment by ptaoussanis** [comment]
As the README mentions- since maintenance was taken over by volunteers in the community, the http-kit.org website is unfortunately no longer maintained and out-of-date.

PR welcome to add an improved client JSON example to the README 👍
